observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemRegistrar.java

```diff
@@ -70,19 +83,21 @@ class MicrometerSubsystemRegistrar implements SubsystemResourceDefinitionRegistr
             "io.micrometer"
     };
 
+    @Deprecated
     public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
             .create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING)
-            .setAttributeGroup(MicrometerConfigurationConstants.OTLP_REGISTRY)
```

Comment:
This is more for a model expert as I don't recall, but is removing an attribute from an attribute group a breaking change? @pferraro @yersan